### PR TITLE
feat: added badge component

### PR DESCRIPTION
**Pull request by sushil-mod**

### CODE REVIEW

File: Components/badge.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+.icon-badge{
+    font-size: 2rem;  
+}
+.badge-on-icon{
+    width: 1.5rem;
+    height: 1.5rem;
+    font-size: 1rem;
+    top: -0.5rem;
+    left:1.8rem;
```
Comment (sks-21) [inline Components/badge.css]: For values less than 10px, rem is not preferred.

File: Components/badge.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+.badge-text-normal{
+    margin-bottom: 2rem;
```
Comment (sks-21) [inline Components/badge.css]: for common properties, you can separate them in the style guide

File: Components/badge.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    font-size: 1.2rem;
+}
+.badge-number{
+    border-radius: 50%;
+    background-color: var(--blue-solid);
+    color: var(--white);
+}
+.badge-text-large{
+    margin-bottom: 2rem;
```
Comment (sks-21) [inline Components/badge.css]: same as above

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: Components/badge.css (hunk 1 above)
Comment (sushil-mod) [reply Components/badge.css]: done, -0.5rem changed to -8px ,thanks

File: Components/badge.css (hunk 2 above)
Comment (sushil-mod) [reply Components/badge.css]: done, removed margin and margin-btm class created in component css and used in badge

File: Components/badge.css (hunk 3 above)
Comment (sushil-mod) [reply Components/badge.css]: done

**Review by ozairkhan2810 — APPROVED** [review]
code looks good, just use formatter before PR

**Pull request merged**